.eslintignore

```diff
@@ -3,3 +3,4 @@ coverage
 docker-compose.yml
 !/.github
 !/docs/.vuepress
+!docs/.vuepress
```

Comment:
Why the line is added again?

> If a glob pattern starts with `/`, the pattern is relative to the base directory of the config file.

Reference: [Ignore Files](https://eslint.org/docs/latest/use/configure/ignore)